experimenter/experimenter/targeting/constants.py

```diff
@@ -87,49 +89,42 @@ def __post_init__(self):
 
 # The following indicate whether the user has changed prefs suggesting
 # they prefer not to see ads or ad-like features
-NEW_TAB_AND_HOMEPAGE_NOT_DEFAULT = """
+NEW_TAB_NOT_DEFAULT = """
 (
-    (
-        !newtabSettings.isDefault
-        ||
-        !'browser.newtabpage.enabled'|preferenceValue
-    )
-    &&
-    !homePageSettings.isDefault
+    !newtabSettings.isDefault
+    ||
+    !'browser.newtabpage.enabled'|preferenceValue
 )
 """
-SPONSORED_SEARCH_SUGGESTIONS_DISABLED = (
-    "!'browser.urlbar.suggest.quicksuggest.sponsored'|preferenceValue"
-)
+HOMEPAGE_NOT_DEFAULT = "!homePageSettings.isDefault"
 TOPSITES_OR_SPONSORED_TOPSITES_DISABLED = """
 (
     !'browser.newtabpage.activity-stream.feeds.topsites'|preferenceValue
     ||
     !'browser.newtabpage.activity-stream.showSponsoredTopSites'|preferenceValue
 )
 """
-SPONSORED_STORIES_DISABLED = """
+RECOMMENDED_OR_SPONSORED_STORIES_DISABLED = """
 (
     !'browser.newtabpage.activity-stream.feeds.section.topstories'|preferenceValue
     ||
     !'browser.newtabpage.activity-stream.showSponsored'|preferenceValue
 )
 """
+SPONSORED_SEARCH_SUGGESTIONS_DISABLED = (
+    "!'browser.urlbar.suggest.quicksuggest.sponsored'|preferenceValue"
+)
 ADS_DISABLED = f"""
 (
-    (
-        {NEW_TAB_AND_HOMEPAGE_NOT_DEFAULT}
-        &&
-        {SPONSORED_SEARCH_SUGGESTIONS_DISABLED}
-    )
+    {NEW_TAB_NOT_DEFAULT}
     ||
-    (
-        {TOPSITES_OR_SPONSORED_TOPSITES_DISABLED}
-        &&
-        {SPONSORED_STORIES_DISABLED}
-        &&
-        {SPONSORED_SEARCH_SUGGESTIONS_DISABLED}
-    )
+    {HOMEPAGE_NOT_DEFAULT}
+    ||
+    {TOPSITES_OR_SPONSORED_TOPSITES_DISABLED}
+    ||
+    {RECOMMENDED_ORO_SPONSORED_STORIES_DISABLED}
```

Comment:
```suggestion
    {RECOMMENDED_OR_SPONSORED_STORIES_DISABLED}
```

I think the above change should make the CI pass.